### PR TITLE
stat schema

### PR DESCRIPTION
**Pull request by siddontang**
we will use show status to show DDL statistics later.

### CODE REVIEW

**Comment by siddontang** [comment]
@ngaut @qiuyesuifeng

File: domain/domain.go [hunk 1]
```diff
@@ -101,6 +103,18 @@ func (do *Domain) SetLease(lease time.Duration) {
...
 }
 
+// Stat returns the DDL statistic.
+func (do *Domain) Stat() (map[string]interface{}, error) {
+	m, err := do.ddl.Stat()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	m["DDL_last_reload_schema_ts"] = atomic.LoadInt64(&do.lastLeaseTS)
```
Comment (qiuyesuifeng) [inline domain/domain.go]: Can unify key of m, both use lower case, like `ddl_last_reload_schema_ts `?
Or like `DDL.LastReloadSchemaTS`?

Comment (siddontang) [inline domain/domain.go]: no, mysql treat the first char to upper. so here just refer mysql.

Comment (siddontang) [inline domain/domain.go]: but I don't like it, I will use lower case later.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**